src/Login.tsx

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import "./App.css";
+import CreateLobbyLink from "./CreateLobbyLink";
+
+interface ILogin {}
+
+export const checkLogin = async () => {
```

Comment:
This isn't checking login as much as it's returning the current user. 

If it were just checking whether or not the current user were logged in, rather than returning a blob with a bunch of the current user's properties, I'd rename it `isLoggedIn` and have it return a boolean instead. But since the places this function is used suggest it's being used to fetch attributes of the current user, I'd rename it `getCurrentUser` and make that intent more clear with some renamings